topology/probes/k8s/probe.go

```diff
@@ -105,3 +106,7 @@ func newClusterLinker(g *graph.Graph, clusterNode *graph.Node, cache *ResourceCa
 	linker := &clusterLinker{graph: g, clusterNode: clusterNode}
 	return graph.NewResourceLinker(g, nil, cache, linker, topology.OwnershipMetadata())
 }
+
+func IsNilValue(i interface{}) bool {
```

Comment:
Why ?

Reply:
if we get a typed nil value from the linker method in k8s.go the we need this function. this was exposed for various configuration as provided in the previous k8s refactor pr

Reply:
@hunchback It happened for one subprobe - and it was caused by a wrong return type of a function that has now been fixed. Do you have an example of such a behaviour ?

Reply:
ok, removed